pom.xml

```diff
@@ -0,0 +1,309 @@
+                                </module>
+                                <module name="UpperEll" />
+                            </module>
+                        </module>
+                    </checkstyleRules>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <groupId>org.apache.rat</groupId>
+                <artifactId>apache-rat-plugin</artifactId>
+                <version>0.12</version>
+                <executions>
+                    <execution>
+                        <id>rat-check</id>
+                        <goals><goal>check</goal></goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <excludes>
+                        <exclude>.travis.yml.*</exclude>
+                        <exclude>**/bnd.bnd</exclude>
+                        <exclude>*.log</exclude>
+                        <exclude>.checkstyle</exclude>
+                        <exclude>.factorypath</exclude>
+                        <exclude>.editorconfig</exclude>
+                        <exclude>**/target/</exclude>
+                        <exclude>**/bin/</exclude>
+                        <exclude>**/*.iml</exclude>
+                        <exclude>**/*.idea</exclude>
+                        <exclude>**/*.ipr</exclude>
+                    </excludes>
+                </configuration>
+            </plugin>
+
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>release</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.sonatype.plugins</groupId>
+                        <artifactId>nexus-staging-maven-plugin</artifactId>
+                        <version>1.6.3</version>
+                        <extensions>true</extensions>
+                        <configuration>
+                            <nexusUrl>https://oss.sonatype.org/</nexusUrl>
+                            <serverId>ossrh</serverId>
+                            <autoReleaseAfterClose>${autorelease}</autoReleaseAfterClose>
+                        </configuration>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-gpg-plugin</artifactId>
+                        <version>1.6</version>
+                        <executions>
+                            <execution>
+                                <id>sign-artifacts</id>
+                                <phase>verify</phase>
+                                <goals>
+                                    <goal>sign</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
```

Comment:
FYI, as you start down the Release process, you may need additional items in your pom.xml to support that process. Not necessary for this initial commit. It looks like you have already referenced existing top-level poms for existing MP component s-- config, fault tolerance, bom, etc. Here are some general instructions for performing a release:
https://wiki.eclipse.org/MicroProfile/SpecRelease/Release

Reply:
Thanks Kevin! I will definitely check this out.